docs/notebooks/ppg/ppg_analysis.ipynb

```diff
@@ -9,9 +9,18 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 73,
```

Comment:
Deze heeft nog een execution count uitstaan. Je kan bovenin de notebook "restart" en "clear_output" klikken, dan is het helemaal opgeschoond.